jamjar/jamjar/videos/serializers.py

```diff
@@ -1,10 +1,13 @@
 from rest_framework import serializers
 from jamjar.videos.models import Video, Edge
-from jamjar.concerts.serializers import ConcertSerializer
+from jamjar.artists.serializers import ArtistSerializer
+from jamjar.users.serializers import UserSerializer
 
 import os
 
 class VideoSerializer(serializers.ModelSerializer):
+    artists = ArtistSerializer(many=True)
```

Comment:
Alright, so this works for the response, but for serializing the request, before we could submit the video with a list of our internal artist ID's.  Now the endpoint will expect a list of _seriailized_ artists, which isn't the desired behavior.